services/hh/processor.go

```diff
@@ -23,7 +23,20 @@ const (
 	bytesWrite  = "bytes_write"
 )
 
+// MetaProvider specifies the interface of a module that the Processor may query
+// to determine active node IDs.
+type MetaProvider interface {
```

Comment:
Ok, I see that the signature differs here than the actual meta store, so you are differentiating that this is not an actual meta store implementation.